docs/pages/reference/cli/tsh.mdx

```diff
@@ -26,6 +26,7 @@ Environment variables configure your tsh client and can help you avoid using fla
 | TELEPORT_USE_LOCAL_SSH_AGENT | Disable or enable local SSH agent integration | true, false |
 | TELEPORT_GLOBAL_TSH_CONFIG | Override location of global `tsh` config file from default `/etc/tsh.yaml` | /opt/teleport/tsh.yaml |
 | TELEPORT_HEADLESS | Use headless authentication | true, false, 1, 0 |
+| TELEPORT_IDENTITY_FILE | File path to identity file | /opt/identity |
```

Comment:
I would indicate what `tsh` uses this for. Is it written to the path when the user authenticates?

Reply:
@ptgott Its a replacement for the -i flag so I mirrored the description there (just "identity file").  This is for calling previously generated credentials (like with machine ID).  How verbose should I get?  And should I update the global flag description too?

Reply:
For me, "identity file" is a little ambiguous, since we use it slightly differently other places in the docs (i.e., a file with TLS and SSH credentials for Teleport API clients). If we're using it differently, I'd add whatever language it takes to disambiguate. If that's not an issue, though, and we define "identity file" in the context of `tsh` elsewhere on the page, we can leave this as is.